upload.js

```diff
@@ -49,8 +50,16 @@ module.exports = function upload () {
     remote = url.format(urlParsed)
   }
 
+  const err = fs.createWriteStream('gk-lockfile-git-push.err')
+
   exec(`git remote add gk-origin ${remote}`)
-  exec(`git push gk-origin HEAD:${info.branchName}`)
+  exec(`git push gk-origin HEAD:${info.branchName}`, {
+    stdio: [
+      'pipe',
+      'pipe',
+      err
```

Comment:
are you sure this works with a write stream? This fails for me

```js
exec(`ls`, {
  stdio: [
    'pipe',
    'pipe',
    fs.createWriteStream('err.our')
  ]
})
// TypeError [ERR_INVALID_OPT_VALUE]: The value "WriteStream {...}" is invalid for option "stdio"
```

This works

```js
exec(`ls`, {
  stdio: [
    'pipe',
    'pipe',
    fs.openSync('err.out', 'w')
  ]
})
```

Reply:
I swear I tried this in a repl, but I can’t reproduce it. Can you send a patch? <3

Reply:
Shall I amend the PR?

Reply:
be my guest :)

Reply:
there you go f8fbbb4, make sure to squash & merge